build.sbt

```diff
@@ -24,15 +24,15 @@ import sbt.Tests.{Group, SubProcess}
 
 val appName = "vat-agent-client-lookup-frontend"
 
-val bootstrapPlayVersion       = "3.3.0"
+val bootstrapPlayVersion       = "4.6.0"
 val govTemplateVersion         = "5.22.0"
-val playPartialsVersion        = "6.1.0"
+val playPartialsVersion        = "6.3.0"
 val authClientVersion          = "2.6.0"
 val playUiVersion              = "7.19.0"
 val playLanguageVersion        = "3.4.0"
 val playWhiteListFilterVersion = "2.0.0"
 val scalaTestPlusVersion       = "2.0.0"
-val hmrcTestVersion            = "3.1.0"
+val hmrcTestVersion            = "3.3.0"
```

Comment:
Could you also update the following:
`scalatest` -> 3.0.5
`scalatestplus-play` -> 2.0.1
`govuk-template` -> 5.23.0
`play-ui` -> 7.22.0
`mockito-core` -> 2.23.4
`jsoup` -> 1.11.3